crates/swc_ecma_parser/src/lexer/jsx.rs

```diff
@@ -157,9 +157,8 @@ impl<'a, I: Input> Lexer<'a, I> {
             if ch == '\\' {
                 has_escape = true;
                 out.push_str(self.input.slice(chunk_start, cur_pos));
-
-                let chars = self.read_jsx_escaped_char(&mut Raw(None))?;
-                out.extend(IntoIterator::into_iter(chars).flatten().flatten());
+                out.push_str("\\");
+                self.bump();
```

Comment:
I think this will break all other valid jsx usages.

Reply:
Hmm...
Tests succeed, but I'm still confused. It makes all tokens handled as `raw`... So I thought it would break things, but seems like it does not?